### PR TITLE
Update navbar

### PR DESCRIPTION
**Pull request by martincai8**

### CODE REVIEW

File: client/src/pages/homepage.js [hunk 1]
```diff
@@ -86,24 +86,6 @@ function Homepage() {
...
-        />
-        <div className="loading-text">
-          <span>Loading</span>
-          <span className="loading-dots">.</span>
-          <span className="loading-dots">.</span>
-          <span className="loading-dots">.</span>
-        </div>
-      </div>
-    );
-  }
```
Comment (michelle-wangg) [inline client/src/pages/homepage.js:105]: was there a reason why this was removed

Comment (martincai8) [reply client/src/pages/homepage.js:105]: i created a new file and overwrote the old one, i'll add the stuff back

Comment (martincai8) [reply client/src/pages/homepage.js:105]: i moved it to App.js; now it only runs during the initial load and not on subsequent homepage visits

**Review by michelleykim — APPROVED** [review]
YOU ABSOLUTE GOAT 
giphy

**Pull request merged**